src/wallet/wallet.cpp

```diff
@@ -1180,6 +1190,10 @@ bool CWallet::AddToWallet(const CWalletTx& wtxIn, bool fFromLoadWallet, CWalletD
         CWalletTx& wtx = (*ret.first).second;
         wtx.BindWallet(this);
         UpdateSproutNullifierNoteMapWithTx(wtx);
+
+        // TODO: Separate function or just use one e.g. UpdateNullifier...
```

Comment:
Yes, this can just be a single function (removing the commit that renamed to `UpdateSproutNullifier...`).